extensions/stats/plugin.cc

```diff
@@ -103,6 +103,19 @@ bool PluginRootContext::onConfigure(size_t) {
           [](const ::Wasm::Common::RequestInfo& request_info) -> uint64_t {
             return request_info.response_size;
           },
+          field_separator, value_separator),
+      StatGen(
+          absl::StrCat(stat_prefix, "tcp_request_bytes"), MetricType::Histogram,
+          [](const ::Wasm::Common::RequestInfo& request_info) -> uint64_t {
+            return request_info.tcp_request_size;
+          },
+          field_separator, value_separator),
+      StatGen(
```

Comment:
Doesn't this produce a label `tcp_request_size` for HTTP requests, too? Should this be guarded/conditioned by a protocol?

Reply:
Will change based on the name we decide above...

Reply:
Removed tcp specific stuff